models/Document/Editable/Area.php

```diff
@@ -79,12 +79,27 @@ public function admin(): void
         $this->outputEditmode('</div>');
     }
 
-    private function renderDialogBoxEditables(array $config, EditableRenderer $editableRenderer, string $dialogId): void
+    private function renderDialogBoxEditables(array|Model\Document\Editable $config, EditableRenderer $editableRenderer, string $dialogId): array
     {
+        if ($config instanceof BlockInterface || $config instanceof Area) {
+            // Unsupported element was passed (e.g., Block, Areablock, ...)
+            // or an Areas was passed, which is not supported to avoid too long editable names
+            $config = [];
```

Comment:
Love the feedback, thanks! I have updated the code, it will now display an error like this:

> Using editables of type "Pimcore\Model\Document\Editable\Area" for the editable dialog "dialogBox-wysiwyg" is not supported. File: /var/www/html/vendor/pimcore/pimcore/models/Document/Editable/Area.php Line: 87